Platform/OpenCanopy/GuiApp.c

```diff
@@ -44,7 +44,9 @@ mLabelNames[LABEL_NUM_TOTAL] = {
   [LABEL_OTHER]                = "Other",
   [LABEL_TOOL]                 = "Tool",
   [LABEL_RESET_NVRAM]          = "ResetNVRAM",
-  [LABEL_SHELL]                = "Shell"
+  [LABEL_SHELL]                = "Shell",
+  [LABEL_SIP_IS_ENABLED]       = "SIPEnabled",
+  [LABEL_SIP_IS_DISABLED]      = "SIPDisabled"
```

Comment:
I have put these labels in a `toggle-sip` branch of `OcBinaryData`.

Reply:
OK.

Reply:
CC @Andrey1970AppleLife